### PR TITLE
chore: add target to build arm64 preflight and use it in vagrant

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

File: vagrant/provision.sh [hunk 1]
```diff
@@ -22,7 +22,19 @@ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scrip
...
+      && rm go.linux-${ARCH}.tar.gz \
+      && ln -s /usr/local/go/bin/go /usr/local/bin
+   cd /sumologic
+   make build_preflight
+else
+   wget "https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/latest/download/preflight-linux-amd64"
+   mv preflight-linux-amd64 preflight
+   chmod +x preflight
+   mv preflight /usr/local/bin/preflight
+fi
```
Comment (sumo-drosiek) [inline vagrant/provision.sh:40]:
```suggestion
fi

```

**Review by swiatekm-sumo — COMMENTED** [review]
It looks like preflight already publishes arm64 builds, can't we use those instead of compiling ourselves?

**Comment by sumo-drosiek** [comment]
I can add it to vagrant, but they doesn't work on darwin 😓

**Comment by swiatekm-sumo** [comment]
> I can add it to vagrant, but they doesn't work on darwin 😓

You mean it doesn't work in a QEMU VM on darwin?

**Comment by sumo-drosiek** [comment]
> You mean it doesn't work in a QEMU VM on darwin?

it doesn't work outside QEMU 😅

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**